src/energy_comms/extract/msa_codes.py

```diff
@@ -0,0 +1,39 @@
+import io
+import pathlib
+
+import numpy as np
+import pandas as pd
+import requests
+
+# not sure why
+# import energy_comms
+
+# from link: https://www.bls.gov/oes/current/msa_def.htm
+
+SOURCE_URL = "https://www.bls.gov/oes/2021/may/area_definitions_m2021.xlsx"
+
+
+def extract():
+    # download area titles and convert to pandas dataframe
+
+    d = pd.read_excel(SOURCE_URL)
+
```

Comment:
Since there's some white space and capital letters in the column names you could do this for uniformity, especially with potentially changing and updating spreadsheets:
```
df.columns = df.columns.str.strip().str.lower().str.replace(" ", "_")
```